TemplateApplication/Schedule/ModalView.swift

```diff
@@ -31,17 +31,17 @@ struct ModalView: View {
                 Text(buttonText)
                     .frame(maxWidth: .infinity, minHeight: 38)
             }
-                .padding()
-                .buttonStyle(.borderedProminent)
+            .padding()
+            .buttonStyle(.borderedProminent)
```

Comment:
I would suggest that we keep this consistent across modifiers (e.g. .padding above is indented) and also keep these indented.

Reply:
Sure! My bad, still used to indent that differently 😄 I'll try to stick to the indented style from now on.